src/Finder/CashOnDeliveryPaymentMethodFinder.php

```diff
@@ -11,17 +11,17 @@
 namespace BitBag\ShopwareInPostPlugin\Finder;
 
 use Shopware\Core\Checkout\Payment\Cart\PaymentHandler\CashPayment;
+use Shopware\Core\Checkout\Payment\DataAbstractionLayer\PaymentMethodRepositoryDecorator;
```

Comment:
This will break in 6.5. Did you debug the container to see if payment_method.repository is there? Or try to fetch it directly from the container and see what happens.